<?xml version="1.0"?>

<launch>

      <node pkg="perception" type="process_camera_image.py" name="process_camera_image">
      <param name="model_path" value="$(find perception)/dd2419_detector_baseline/trained_models/det_2021-02-16_15-14-19-181578.pt"/>
      <param name="annotation_path" value="$(find perception)/dd2419_detector_baseline/dd2419_coco/annotations/training.json"/>  
      </node>
        

</launch>